extensions/cornerstone-dicom-seg/src/getPanelModule.tsx

```diff
@@ -1,7 +1,8 @@
 import React from 'react';
 
 import { useAppConfig } from '@state';
-import { Toolbox } from '@ohif/ui';
+import { Toolbox as NewToolbox } from '@ohif/ui-next';
```

Comment:
Add a comment to the use of this that it is activated by the use experimental flag and that the use of OldToolBox is going away once the new one is fully functional.